plugins/conf_maps/lib/api/KalturaConfMaps.php

```diff
@@ -105,17 +110,41 @@ public function validateForInsert($propertiesToSkip = array())
 		parent::validateForInsert($propertiesToSkip);
 	}
 
-	public function validateContent()
+	public function validateAndHandleContent()
```

Comment:
Other than validating the content , does this function has any side effect? if not then leave it original name